subsystem-arm/test/hand/hand-modes_test.cpp

```diff
@@ -0,0 +1,64 @@
+#include "testing/testing_frameworks.hpp"
+#include "utility/log.hpp"
+#include "../implementations/hand/hand-modes.hpp"
+#include "../dto/arm-dto.hpp"
+#include "../implementations/pca9685.hpp"
+
+
+namespace sjsu::arm
+{
+    TEST_CASE("Hand Mode Testing")
+    {
+        SECTION("1.1 Should set angles to correct given individual mode angles")
+        {
+            hand_arguments original_commands{100, 100, 100, 100, 100};
+            hand_arguments args = HandModes::IndividualMode(original_commands);
+            CHECK_EQ(args.pinky_angle, original_commands.pinky_angle);
+            CHECK_EQ(args.ring_angle, original_commands.ring_angle);
+            CHECK_EQ(args.middle_angle, original_commands.middle_angle);
+            CHECK_EQ(args.index_angle, original_commands.index_angle);
+            CHECK_EQ(args.thumb_angle, original_commands.thumb_angle);
+        }
+
+        SECTION("2.1 Should set angles to minimum angle when in close mode")
+        {
+            hand_arguments original_commands{30, 30, 30, 30, 30};
```

Comment:
```suggestion
            hand_arguments original_commands{30, 30, 30, 30, 30};
```
Swap out the random special numbers with the static constexpr variables

Reply:
do the same with the other special numbers